recipes/wordnut

```diff
@@ -0,0 +1,3 @@
+(wordnut :repo "gromnitsky/wordnut"
+	:fetcher github
+	:files ("*.el"))
```

Comment:
You need not specify `:files` property. `*.el` is default value.

Please see item `:files` of https://github.com/milkypostman/melpa#recipe-format